WireDomain/Project/WireDomain Project.xcodeproj/project.pbxproj

```diff
@@ -630,6 +622,7 @@
 			);
 			mainGroup = 01D0DC9C2C1C8C870076CB1C;
 			packageReferences = (
+				EE463E0D2C7360BD0089C78B /* XCLocalSwiftPackageReference "../../WireUtilities" */,
```

Comment:
Please don't use the "Package Dependencies" UI for linking to Swift packages of the same workspace.
I merged a PR recently fixing this: https://github.com/wireapp/wire-ios/pull/1761

It causes redundancy and issues when the frameworks are copied into the bundle at the end of the build process (leading to the same framework being added more than once).
Also notice, that the dialog for linking frameworks shows the same target twice in your branch:
<img width="583" alt="Screenshot 2024-08-20 at 1 27 50 PM" src="https://github.com/user-attachments/assets/ac311c51-d2f0-46b7-acbb-b924fff27b1a">

Reply:
Ah ok I didn't know, thanks for the heads up